core/src/banking_stage/consumer.rs

```diff
@@ -545,6 +558,19 @@ impl Consumer {
         });
         execute_and_commit_timings.collect_balances_us = collect_balances_us;
 
+        let min_max = batch
+            .sanitized_transactions()
+            .iter()
+            .filter_map(|transaction| {
+                let round_compute_unit_price_enabled = false; // TODO get from working_bank.feature_set
+                transaction
+                    .get_transaction_priority_details(round_compute_unit_price_enabled)
```

Comment:
this is scanning through all transaction ixs to get the priority. @taozhu-chicago - this is another case motivating the `RuntimeTransaction` that carries meta 😢